platform/ext/target/nordic_nrf/common/nrf9160/target_cfg.c

```diff
@@ -533,7 +533,7 @@ enum tfm_plat_err_t spu_init_cfg(void)
      * in case this has been overwritten by earlier images e.g.
      * bootloader.
      */
-    spu_regions_reset_all_secure();
+    spu_regions_reset_unlocked_secure();
```

Comment:
The comment above here doesn't make sense anymore.

(I realize it was wrong before, the code was trying to do something and failing, now we are trying to do something else, and the comment must reflect what we are now trying to do)